lib/events.js

```diff
@@ -12,12 +12,19 @@ var fxaEvents;
 
 if (!config.events.region || !config.events.queueUrl) {
   fxaEvents = {
-    start: function () { logger.warn('accountEvent.unconfigured'); }
+    start: function start() {
+      if (config.env === 'prod') {
```

Comment:
We have `  var isProd = config.env === 'prod';` in other others of the code, should we update those to include stage as well while we think of it?